frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -160,10 +169,15 @@ export class Wizard {
    * @param   {string}              name     Name of a disk to edit.
    * @param   {Disk}     disk     Disk with the requested attributes.
    */
-  async editDisk(name: string, disk: Disk) {
+  async editDISK(name: string, disk: Disk) {
```

Comment:
why DISK?

Reply:
Cause I have method `editDisk` in a child class and it has different signature and I've got a conflict. So I changed a name to make it a different method to avoid that conflict. In `vmwareImportWizard` I don't need to send name of the disk and disk separately

Reply:
Fixed